content/token-program.md

```diff
@@ -340,6 +345,8 @@ async function buildTransferTransaction(
 }
 ```
 
+JC: in the name of encouraging associated token accounts, this could take in the destination wallet address and use `getOrCreateAssociatedTokenAccount` to get the destination token account
```

Comment:
@joncinque this comment is in reference to the code snippet above, correct? If so, this snippet is only meant to show the implementation of the `spl-token` library's `transfer` function, which from what I can tell assumes that the address being passed into the function is already the correct token account